components/Home/RecurringEvents.vue

```diff
@@ -1,19 +1,22 @@
 <i18n lang="yaml">
 en:
   announcement: <strong>DWH</strong> is open {count} nights a week
+  announcement-month: Other monthly events at <strong>DWH</strong>
```

Comment:
```suggestion
  announcement_month: Other monthly events at <strong>DWH</strong>
```
for consistency (below as well)